packages/material-ui/src/AccordionSummary/AccordionSummary.test.js

```diff
@@ -99,19 +111,34 @@ describe('<AccordionSummary />', () => {
     expect(button).not.to.have.class(classes.focused);
   });
 
+  it('should fire onBlur when the button blurs', () => {
+    const handleBlur = spy();
+    const { getByRole } = render(<AccordionSummary onBlur={handleBlur} />);
+
+    act(() => {
+      const button = getByRole('button');
+      button.focus();
+      button.blur();
+    });
+
+    expect(handleBlur.callCount).to.equal(1);
+  });
+
   it('should fire onClick callbacks', () => {
     const handleClick = spy();
     const { getByRole } = render(<AccordionSummary onClick={handleClick} />);
 
-    getByRole('button').click();
+    act(() => {
```

Comment:
Does this fail without the `act`? If not I'd prefer to use `act` needlessly just so that we're more aware of how changes affect tests.

Reply:
It doesn't fail without act. Should I revert my change?

Reply:
That'd be nice. I'm not sure yet if we actually need it when testing in concurrent mode and I'd rather see the explicit fail when we switch.

Reply:
OK great, I actually prefer this approach too.

Reply:
It wasn’t failing without `act` for me.